src/common/modals/Settings/components/General.js

```diff
@@ -530,7 +534,18 @@ const General = () => {
         </p>
         <Switch
           onChange={e => {
+            if (!potatoPcMode) {
+              dispatch(
+                openModal('ActionConfirmation', {
+                  message:
+                    'The computer needs to be restarted for this operation',
```

Comment:
The wording here is bad. I think you should say "Restart GDLauncher to complete this operation?" or something like that, instead of implying the user should restart their whole computer

Reply:
yeah my bad